_api-reference/cat/cat-shards.md

```diff
@@ -33,6 +33,7 @@ Parameter | Type | Description
 bytes | Byte size | Specify the units for byte size. For example, `7kb` or `6gb`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
 local | Boolean | Whether to return information from the local node only instead of from the cluster manager node. Default is `false`.
 cluster_manager_timeout | Time | The amount of time to wait for a connection to the cluster manager node. Default is 30 seconds.
+cancel_after_time_interval | Time | The time after which the shards request will be canceled. Default is -1.
```

Comment:
```suggestion
cancel_after_time_interval | Time | The amount of time after which the shard request will be canceled. Default is `-1`.
```

Reply:
I was referring the flag from the cat search API, Can you please check that - https://opensearch.org/docs/latest/api-reference/search/
If you still want i will make above suggestion.

Reply:
This way it's clearer that it's the amount of time and not just a time on the clock. We should change in search API too.